src/client/index.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
 import { render } from 'react-dom'
-import { Router, match, browserHistory } from 'react-router'
+import { match, browserHistory } from 'react-router'
 import AsyncProps from 'async-props'
 import 'location-origin'
 import 'es6-promise/auto'
 import RouteWrapper from '../shared/route-wrapper'
 import config from 'tapestry.config.js'
 import mitt from 'mitt'
+import { AppContainer } from 'react-hot-loader'
+import Root from './root'
```

Comment:
Is it worth exploring a way to keep `AppContainer`/`react-hot-loader` out of our production bundle so we don't ship it?

Could look into using the `__DEV__` flag, and the minification process should strip this out?

There may be more than one strategy: https://stackoverflow.com/questions/29795782/exclude-a-module-from-webpack-bundle